.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
+//registry.npmjs.org/:_authToken=1b1227dc-c4df-4d00-abf0-bf4549c98140
```

Comment:
this constitutes a password; you will want to change your npm password IMMEDIATELY.